agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2011,47 +2011,61 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
     // which is currently not supported. See "Send standard NL80211 commands using DWPAL #782"
```

Comment:
Commit 9fe1245e3381926fb5846b87f4ef0f1951c2d59b should be squashed to commit dedefbf. Also, do not put the commit numbers in the commit message unless you're sure you won't change it again, cause if you force push this commit will be deleted.

Reply:
squashed